products/jbrowse-cli/src/commands/admin-server.test.ts

```diff
@@ -0,0 +1,216 @@
+/**
+ * @jest-environment node
+ */
+
+import fs from 'fs'
+import * as path from 'path'
+import fetch from 'node-fetch'
+import { Context } from 'vm'
+import { setup } from '../testUtil'
+
+const fsPromises = fs.promises
+
+const defaultConfig = {
+  assemblies: [],
+  configuration: {},
+  connections: [],
+  defaultSession: {
+    name: 'New Session',
+  },
+  tracks: [],
+}
+
+const testConfig = path.join(
+  __dirname,
+  '..',
+  '..',
+  'test',
+  'data',
+  'test_config.json',
+)
+
+const testConfigContents = {
+  assemblies: [
+    {
+      name: 'testAssembly',
+      sequence: {
+        type: 'testSequenceTrack',
+        trackId: '',
+        adapter: {
+          type: 'testSeqAdapter',
+          twoBitLocation: {
+            uri: 'test.2bit',
+          },
+        },
+      },
+    },
+  ],
+  configuration: {},
+  connections: [],
+  defaultSession: {
+    name: 'New Session',
+  },
+  tracks: [],
+}
+
+// extend setup to include the addition of a simple HTML index to serve statically
+const testIndex = path.join(
+  __dirname,
+  '..',
+  '..',
+  'test',
+  'data',
+  'simpleIndex.html',
+)
+const setupWithCreate = setup.do(async ctx => {
+  await fsPromises.copyFile(
+    testIndex,
+    path.join(ctx.dir, path.basename(testIndex)),
+  )
+})
+
+async function killExpress(ctx: Context, port: number) {
```

Comment:
Can just define the type inline here

```suggestion
async function killExpress(ctx: { stdoutWrite: jest.Mock }, port: number) {
```